bin/node/runtime/Cargo.toml

```diff
@@ -14,59 +14,59 @@ rustc-hex = { version = "2.0", optional = true }
-pallet-finality-tracker = { path = "../../../frame/finality-tracker", default-features = false }
-pallet-grandpa = { path = "../../../frame/grandpa", default-features = false }
-pallet-im-online = { path = "../../../frame/im-online", default-features = false }
-pallet-indices = { path = "../../../frame/indices", default-features = false }
-pallet-membership = { path = "../../../frame/membership", default-features = false }
-pallet-nicks = { path = "../../../frame/nicks", default-features = false }
-pallet-offences = { path = "../../../frame/offences", default-features = false }
-pallet-randomness-collective-flip = { path = "../../../frame/randomness-collective-flip", default-features = false }
-pallet-session = { path = "../../../frame/session", default-features = false, features = ["historical"] }
-pallet-staking = { path = "../../../frame/staking", default-features = false, features = ["migrate"] }
+pallet-authority-discovery = { version = "2.0.0", default-features = false, path = "../../../frame/authority-discovery" }
+pallet-authorship = { version = "2.0.0", default-features = false, path = "../../../frame/authorship" }
+pallet-babe = { version = "0.8", default-features = false, path = "../../../frame/babe" }
+pallet-balances = { version = "2.0.0", default-features = false, path = "../../../frame/balances" }
+pallet-collective = { version = "2.0.0", default-features = false, path = "../../../frame/collective" }
+pallet-contracts = { version = "2.0.0", default-features = false, path = "../../../frame/contracts" }
+pallet-contracts-rpc-runtime-api = { version = "2.0.0", default-features = false, path = "../../../frame/contracts/rpc/runtime-api/" }
+pallet-democracy = { version = "2.0.0", default-features = false, path = "../../../frame/democracy" }
+pallet-elections-phragmen = { version = "2.0.0", default-features = false, path = "../../../frame/elections-phragmen" }
+frame-executive = { version = "2.0.0", default-features = false, path = "../../../frame/executive" }
+pallet-finality-tracker = { version = "2.0.0", default-features = false, path = "../../../frame/finality-tracker" }
+pallet-grandpa = { version = "2.0.0", default-features = false, path = "../../../frame/grandpa" }
+pallet-im-online = { version = "2.0.0", default-features = false, path = "../../../frame/im-online" }
+pallet-indices = { version = "2.0.0", default-features = false, path = "../../../frame/indices" }
+pallet-membership = { version = "2.0.0", default-features = false, path = "../../../frame/membership" }
+pallet-nicks = { version = "2.0.0", default-features = false, path = "../../../frame/nicks" }
+pallet-offences = { version = "2.0.0", default-features = false, path = "../../../frame/offences" }
+pallet-randomness-collective-flip = { version = "2.0.0", default-features = false, path = "../../../frame/randomness-collective-flip" }
+pallet-session = { features = ["historical"], path = "../../../frame/session", default-features = false }
```

Comment:
Why no version here?

Reply:
that's a good question. I was doing a regexp-replace, it appears it didn't catch all cases.. need to figure out why and which ones...